recipes/libabigail/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "libabigail" %}
+{% set version = "2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://mirrors.kernel.org/sourceware/{{ name }}/{{ name }}-{{ version }}.tar.xz
+  sha256: 7cfc4e9b00ae38d87fb0c63beabb32b9cbf9ce410e52ceeb5ad5b3c5beb111f3
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - make
+  host:
+    - elfutils
+  run:
+    - libxml2
+
+test:
+  commands:
```

Comment:
Shouldn't this also contain some libraries one links to? Please check for their existence here and add a run_exports.

Reply:
Thanks, you're right, I didn't know about this field.  The library number changes every minor release, so I think I added this correctly...